lib/horde/supervisor_impl.ex

```diff
@@ -218,9 +194,15 @@ defmodule Horde.SupervisorImpl do
     {:noreply, mark_alive(state, true)}
   end
 
-  defp get_child_pid(%{name: name}, child_id) do
-    [{pid, _}] = Registry.lookup(registry_name(name), child_id)
-    pid
+  def handle_cast({:remove_process_tracking, child_id}, state) do
```

Comment:
Perhaps we can come up with a better name for this than `remove_process_tracking`. Something that indicates its intent (ie, the supervisor has indicated that the child has terminated and should not be restarted).

Reply:
maybe aborted_child?

Reply:
Or another would be forget_child.

Reply:
What do you think?